Bottom: 5b76a802a8d39680ba270d27ab42350260065e53
Top:    ed58a354edeb7dd8b911c0f31c86466d5f750f5b
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-06-13 01:22:41 +0100

Fix some broken assumptions regarding 32-bit integer coordinates in polygon1.c


---

diff --git a/src/polygon1.c b/src/polygon1.c
index 572a138..1cf24db 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -2511,8 +2511,8 @@ poly_IniContour (PLINE * c)
   /* bzero (c, sizeof(PLINE)); */
   c->head.next = c->head.prev = &c->head;
 #warning BROKEN ASSUMPTION of 32 BIT
-  c->xmin = c->ymin = 0x7fffffff;
-  c->xmax = c->ymax = 0x80000000;
+  c->xmin = c->ymin = COORD_MAX;
+  c->xmax = c->ymax = -COORD_MAX - 1;
   c->is_round = FALSE;
   c->cx = 0;
   c->cy = 0;
